engine/src/main/java/org/terasology/rendering/shader/ShaderParametersCombine.java

```diff
@@ -107,6 +107,12 @@ public void applyParameters(Material program) {
             program.setFloat("outlineThickness", outlineThickness, true);
         }
 
+        //TODO: Other parameters and volumetric fog test case is needed
+        Camera activeCamera = CoreRegistry.get(WorldRenderer.class).getActiveCamera();
```

Comment:
Throughout the implementation, I felt that lines with _"CoreRegistry.get(Config.class).getRendering()"_ are in a style of convention. After hearing this, I will definitely refactor it.